src/SystemAbstractionsTests/MicrosoftUnitTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.SystemAbstractions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Win32;
+using Moq;
+
+namespace SystemAbstractionsTests
+{
+[TestClass]
+    public class MicrosoftUnitTests
+    {
+        const string WindowsVersionRegKey = @"HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion";
```

Comment:
>const [](start = 8, length = 5)

Seems odd to have a class-level const that only gets used in 1 method. I'd suggest creating 2 consts (WindowsVersionRegKey and CurrentVersion) inside the test method, then passing the const values to both the "direct" and "indirect" methods being called

Reply:
It is used in two methods.

Reply:
Done.